templates/article.html

```diff
@@ -130,8 +131,21 @@ <h2 class="card-title">{{ article.title }}</h2><!--title-->
           <textarea class="form-control" rows="3" name="body" placeholder="write something here..." required="required"></textarea>
         </div>
 
-        <button class="btn btn-primary" type="submit" style="float:right">comment</button>
+        <button id="btn" class="btn btn-primary" type="submit" style="float:right"  onClick="prevent()">comment</button>
     </form>
+    <script type="text/javascript">
+        // 第一种 通过点击事件
+        var clicktag = 1;
+        btn.onclick = function () {
+            if (clicktag == 3) {
+                alert("请勿频繁点击！");
+                setTimeout(function () { clicktag = 0 }, 5000);
```

Comment:
@1500h 

Probably you want to use a bigger number than 5000, as a more serious warning.

-Hui